zerver/tests/test_signup.py

```diff
@@ -699,6 +699,21 @@ def test_successful_invite_user(self) -> None:
         self.assertTrue(find_key_by_email(invitee))
         self.check_sent_emails([invitee], custom_from_name="Hamlet")
 
+        prereg_user = PreregistrationUser.objects.get(email=invitee)
+        url = self.get_confirmation_url_from_outbox(invitee)
+        # should have no effect for non-staff users
+        url += "?support=True"
+
+        self.assertEqual(prereg_user.status, 0)
+        response = self.client_get(url)
+        prereg_user.refresh_from_db()
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(prereg_user.status, 1)
+
+        self.submit_reg_form_for_user(invitee, "password")
+        new_user = UserProfile.objects.get(email="alice-test@zulip.com")
+        self.assert_logged_in_user_id(new_user.id)
+
```

Comment:
Isn't this covered by ``test_successful_invite_with_support_param`` now, and thus just pure duplication that we can remove?

Reply:
Ah, on second look I see this is testing the scenario "logged in as normal user", which isn't duplicate. This should be moved into ``test_successful_invite_with_support_param`` for consistency then (or maybe split that function into 3 separate ``test_..._as_normal_user``, ``test_..._as_admin``, ``test_..._logged_out`` or something)